app/editor/src/features/content/form/Breadcrumb.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+import * as styled from './styled';
+export interface IBreadcrumbProps {
+  /** the current path being viewed in the clip explorer */
+  path: string;
+  /** set path of the parent component */
+  setPath: (path: string) => void;
+}
+
+/** component used to navigate state controlled path directory, also updates the
+ *  search params to the current directory
+ */
+export const Breadcrumb: React.FC<IBreadcrumbProps> = ({ path, setPath }) => {
```

Comment:
Should this be a component not a feature?  `./components/breadcrumb`?  
Remember `features` are for business functionality while `components` are reusable tools that we could put in multiple different projects.

Reply:
I was actually going to ask if it is general enough to move to tno-core? Guessing maybe it would need some work before that though